server/src/apub/activities.rs

```diff
@@ -57,16 +56,18 @@ where
   for t in to {
     let to_url = Url::parse(&t)?;
     if !is_apub_id_valid(&to_url) {
-      debug!("Not sending activity to {} (invalid or blocklisted)", t);
+      debug!("Not sending activity to {} (invalid or blacklisted)", t);
       continue;
     }
-    let request = Request::post(t).header("Host", to_url.domain().unwrap());
-    let signature = sign(&request, actor)?;
+    let mut request = attohttpc::post(t).header("Host", to_url.domain().unwrap());
+    let signature = sign(&mut request, actor)?;
```

Comment:
It would be nice if the sign method could just take in the url, and actor, that way you could just build the actual request afterwards.

Reply:
Well it has to sign/verify the HTTP headers, so you would have to pass them in anyway. I dont think it would simplify anything.